src/jit/gentree.cpp

```diff
@@ -1791,6 +1791,11 @@ regMaskTP GenTreeCall::GetOtherRegMask() const
     return resultMask;
 }
 
+bool GenTreeCall::IsPure(Compiler* compiler) const
+{
```

Comment:
Should this be called IsPureHelper instead?
Someday we could determine if a Jitted method being called is Pure